app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,33 @@
+class NotificationMailer < ApplicationMailer
+  helper :application
+
+  before_action :set_inline_attachments
+
+  def digest
+    @user = params[:user]
+    @notifications = params[:notifications]
+    @type = params[:type]
+
+    set_user_avatar
+
+    count = @notifications.count
+    mail to: @user.email, subject: "You have #{count} unread #{'notification'.pluralize(count)}"
+  end
+
+  private
+
+  def find_asset(name)
+    Rails.application.assets.find_asset(name).pathname
+  end
+
+  def set_inline_attachments
+    attachments.inline['profile'] = File.read(find_asset('profile.jpg'))
+    attachments.inline['logo_small'] = File.read(find_asset('logo_small.png'))
+    attachments.inline['DradisCE_full_small'] = File.read(find_asset('DradisCE_full_small.png'))
+  end
+
+  def set_user_avatar
+    # The arguments are a noop here for CE-Pro parity.
+    @avatar_url = attachments['profile'].url
```

Comment:
`avatar_url` is always set to the attached profile image and won't actually attempt to load the gravatar. CE users should also receive the gravatar if it's available.

Reply:
I didn't implement that logic because we don't have working gravatars in CE yet (see: [application_helper.rb](https://github.com/dradis/dradis-ce/blob/notifications-mailer/app/helpers/application_helper.rb#L2)). What you're pointing out should be the correct behavior, though.

Reply:
It's been updated in master:
https://github.com/dradis/dradis-ce/blob/master/app/helpers/avatar_helper.rb

Reply:
Updated!